src/extension/notifier/src/main/resources/applicationContext.xml

```diff
@@ -4,6 +4,8 @@
     <bean id="notifierProcess" class="au.org.emii.wps.NotifierProcess">
         <constructor-arg index="0" ref="wpsResourceManager"/>
         <constructor-arg index="1" ref="httpNotifier"/>
+        <constructor-arg index="2" value="3" />
+        <constructor-arg index="3" value="300000" />
```

Comment:
May I suggest injecting that as a property? Then it'll have a name and be more self-describing. See also:
https://github.com/aodn/geoserver-build/blob/ncwms_wps_plugin/src/extension/ncwms/src/main/java/applicationContext.xml#L8

*The presenter is not a spring expert*